cfme/control/explorer/conditions.py

```diff
@@ -271,6 +273,9 @@ def create(self, condition_class, description, expression=None, scope=None, note
         view.add_button.click()
         view = condition.create_view(ConditionDetailsView, wait="10s")
         view.flash.assert_success_message('Condition "{}" was added'.format(condition.description))
+        if BZ(1683697, forced_streams=["5.9", "5.10"]).blocks:
+            self.appliance.browser.widgetastic.browser.refresh()
```

Comment:
Can you put a comment on why you are refreshing here? Does forcing navigation not work?

Reply:
Forcing navigation would actually work -- I will make this change.